pkg/test/resourcefixture/testdata/basic/container/v1beta1/containercluster/containercluster-autoscaling-surgeupgrade/_http.log

```diff
@@ -638,7 +650,7 @@ X-Xss-Protection: 0
   "initialClusterVersion": "1.30.5-gke.1014001",
   "initialNodeCount": 1,
   "instanceGroupUrls": [
-    "https://www.googleapis.com/compute/v1/projects/${projectId}/zones/us-central1-a/instanceGroupManagers/gke-containercluster-abcdef"
+    "https://www.googleapis.com/compute/v1/projects/${projectId}/zones/us-west1-b/instanceGroupManagers/gke-containercluster-abcdef"
```

Comment:
Is a complete zone change expected here? us-central1-a -> us-west1-b

Reply:
This is a fix. Previously we blindly used `us-central1-a` in the mock.